### PR TITLE
Feature/inquiry sample expansion

### PR DESCRIPTION
**Pull request by pnobbe**
### Merge when:

* [x] The merge build succeeded
* [ ] The PR has been reviewed and is approved
* [ ] All packages are upgraded to the latest version (only PR into master)

### What issue does this PR address?

This PR includes the necessary changes for our inquiry expansion.

### Does this PR introduce a breaking change?

Yes:
- `BaseMeasurementLevel` property is removed from the `InquirySample` entity. A database migration should be done to delete its respective column entry from all database instances.
- `SkewedWindowFrame` property has been added to the `InquirySample` entity. A database migration should be done to add its respective column entry to all database instances.
-  `FoundationDamageCause` property has been made optional in the `InquirySample` entity. A database migration should be done to change its respective column definition in all database instances. Relates to #489 
- The `Unknown` value in the `FoundationDamageCause` enum has been removed from the enum declaration in favor of declaring it `null` in the instance where `Unknown` was previously used. Any existing database entries where it's value (7) exists should be set to `null` instead.
- The `Unknown` value in the `FoundationType` enum has been removed from the enum declaration in favor of declaring it `null` in the instance where `Unknown` was previously used. Any existing database entries where it's value (15) exists should be set to `null` instead.

### Additional information

* [x] Remove branch after merge (only features into develop)

### CODE REVIEW

**Comment by pnobbe** [comment]
Partially fixes #489

File: src/FunderMaps.Core/Entities/InquirySample.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿using FunderMaps.Core.Types;
+using FunderMaps.Core.Types;
```
Comment (yorickdewid) [inline src/FunderMaps.Core/Entities/InquirySample.cs:1]: What happened here?

File: src/FunderMaps.Core/Types/FoundationDamageCause.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿namespace FunderMaps.Core.Types
+namespace FunderMaps.Core.Types
```
Comment (yorickdewid) [inline src/FunderMaps.Core/Types/FoundationDamageCause.cs:1]: Again? Whats this?

File: src/FunderMaps.Core/Types/FoundationType.cs [hunk 3]
```diff
@@ -78,11 +78,6 @@ public enum FoundationType
         /// <summary>
         ///     Other.
         /// </summary>
-        Other = 14,
-
-        /// <summary>
-        ///     Unknown.
-        /// </summary>
-        Unknown = 15,
+        Other = 14
```
Comment (yorickdewid) [inline src/FunderMaps.Core/Types/FoundationType.cs:81]: Leave trailing comma. That would not have marked this line as 'changed'

File: src/FunderMaps.Data/Repositories/InquirySampleRepository.cs [hunk 4]
```diff
@@ -300,64 +303,64 @@ private static InquirySample MapFromReader(DbDataReader reader, int offset = 0)
...
+                CrackFacadeRightSize = reader.GetSafeDecimal(offset + 55),
+                DeformedFacade = reader.GetSafeBoolean(offset + 56),
+                ThresholdUpdownSkewed = reader.GetSafeBoolean(offset + 57),
+                ThresholdFrontLevel = reader.GetSafeDecimal(offset + 58),
+                ThresholdBackLevel = reader.GetSafeDecimal(offset + 59),
+                SkewedParallel = reader.GetSafeDecimal(offset + 60),
+                SkewedPerpendicular = reader.GetSafeDecimal(offset + 61),
+                SkewedFacade = reader.GetFieldValue<RotationType?>(offset + 62),
+                SettlementSpeed = reader.GetSafeDouble(offset + 63),
+                SkewedWindowFrame = reader.GetSafeBoolean(offset + 64)
```
Comment (yorickdewid) [inline src/FunderMaps.Data/Repositories/InquirySampleRepository.cs:363]: Always trailing comma if supported by the language

**Review by yorickdewid — CHANGES_REQUESTED** [review]
See the comments

File: src/FunderMaps.Core/Entities/InquirySample.cs (hunk 1 above)
Comment (yorickdewid) [reply src/FunderMaps.Core/Entities/InquirySample.cs:1]: UTF w/o BOM

Comment (pnobbe) [reply src/FunderMaps.Core/Entities/InquirySample.cs:1]: Files on origin were encoded in UTF-8 with BOM. This changes the file to UTF-8 without BOM.

**Review by yorickdewid — APPROVED** [review]

File: src/FunderMaps.Core/Types/FoundationDamageCause.cs (hunk 2 above)
Comment (pnobbe) [reply src/FunderMaps.Core/Types/FoundationDamageCause.cs:1]: Files on origin were encoded in UTF-8 with BOM. This changes the file to UTF-8 without BOM.

**Pull request merged**